esmvalcore/preprocessor/_regrid_esmpy.py

```diff
@@ -243,7 +243,10 @@ def get_grid(esmpy_lat, esmpy_lon,
         num_peri_dims = 1
     else:
         num_peri_dims = 0
-    grid = esmpy.Grid(np.array(esmpy_lat.shape),
+
+    # previous to numpy 2.0, a np.array(esmpy_lat.shape) was used
+    # numpy>=2.0 throws ValueError: matrix transpose with ndim<2 is undefined
+    grid = esmpy.Grid(np.vstack(esmpy_lat.shape),
```

Comment:
Is this backwards-compatible? If yes, do we need this comment?

Reply:
yes, it is, functionally and it returns the same sort of structure, only typed differently